moveit_planners/trajopt/trajopt/README.md

```diff
@@ -0,0 +1,3 @@
+TODO(ommmid): Remove code duplication.
+This package temporarily copied into moveit until the repo trajopt_ros does not depend on Tesseract
+See ros-industrial-consortium/trajopt_ros#122
```

Comment:
I'd just delete this file.

Reply:
I requested this file a while back. Are you thinking we won't merge back to TrajOpt's other repo? If we do plan on doing it eventually, I don't see how this documentation hurts anything...

Reply:
To me it makes more sense to keep trajopt in MoveIt and assume maintainership. I don't feely strongly about this. That's why I approved the PR as-is.